<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubwayDemoClass</class>
 <widget class="QMainWindow" name="SubwayDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>上海地铁换乘指南</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="7,2">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>地铁线路图</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,0,15">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>出发站</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lin_start"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>到达站</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lin_end"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pbtn_clr">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>清除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbtn_ok">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbtn_rfs">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>换乘方案</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextBrowser" name="txb_rst"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>管理员界面</string>
    </property>
    <addaction name="actionadd"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionadd">
   <property name="text">
    <string>添加线路</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SubwayDemo.qrc"/>
 </resources>
 <connections/>
</ui>
